cmd/kube-scheduler/app/server.go

```diff
@@ -259,6 +260,7 @@ func newMetricsHandler(config *kubeschedulerconfig.KubeSchedulerConfiguration) h
 			goruntime.SetBlockProfileRate(1)
 		}
 	}
+	installLogLevelHandler(pathRecorderMux)
```

Comment:
We should do that universally for all components - it will be super useful.

Reply:
Your're right! I will install the handler for `controller-manager` as well.

Reply:
@wojtek-t now support for `controller-manager` as well, and kubelet, apiserver are already enabled by #63777 and #64601.